<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="QLabel_Measurement">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>400</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquare</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Icon/Text_measurement.png);</string>
   </property>
   <property name="text">
    <string>Measurement</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Measurement">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-image: url(:/Icon/measurement.png);
}

QPushButton:pressed {
	border-image: url(:/Icon/measurement_pressed.png);
}



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Data">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-image: url(:/Icon/data.png);
}

QPushButton:pressed {
	border-image: url(:/Icon/data_pressed.png);
}



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Setting">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>180</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-image: url(:/Icon/setting.png);
}

QPushButton:pressed {
	border-image: url(:/Icon/setting_pressed.png);
}



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Shutdown">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>520</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-image: url(:/Icon/Shutdown.png);
}

QPushButton:pressed {
	border-image: url(:/Icon/Shutdown_pressed.png);
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Reset">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>520</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-image: url(:/Icon/back_icon.png);
}

QPushButton:pressed {
	border-image: url(:/Icon/back_pressed.png);
}



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_EasterEgg">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Icon/logo.png);
font: 1pt &quot;Ubuntu&quot;;


</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_mini">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>1</width>
     <height>1</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Backspace">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-image: url(:/Icon/backspace.png);
}

QPushButton:pressed {
	border-image: url(:/Icon/backspace_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Label_date">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>35</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquare</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>20-06-06</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Label_time">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>35</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>NanumSquare</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AM 8:30:58</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Label_USB">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>30</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>40</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquare</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>℃</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Label_temp3">
   <property name="geometry">
    <rect>
     <x>941</x>
     <y>40</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquare</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>33</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Label_temp1">
   <property name="geometry">
    <rect>
     <x>825</x>
     <y>40</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquare</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>33</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>854</x>
     <y>40</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquare</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>℃</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Label_temp3_2">
   <property name="geometry">
    <rect>
     <x>782</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum square</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Nanum square&quot;;
color: #a3a3a3;</string>
   </property>
   <property name="text">
    <string>Cur.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Label_temp3_3">
   <property name="geometry">
    <rect>
     <x>899</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum square</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Nanum square&quot;;
color: #a3a3a3;</string>
   </property>
   <property name="text">
    <string>Set.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="QLabel_Measurement_2">
   <property name="geometry">
    <rect>
     <x>436</x>
     <y>400</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquare</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Icon/Text_measurement.png);</string>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="QLabel_Measurement_3">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>400</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumSquare</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Icon/Text_measurement.png);</string>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Label_USB_2">
   <property name="geometry">
    <rect>
     <x>732</x>
     <y>20</y>
     <width>30</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Icon/temperature2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Icon/Icon.qrc"/>
 </resources>
 <connections/>
</ui>
